tests/functional/firmware/k21-rtt/src/main.c

```diff
@@ -0,0 +1,57 @@
+/******************************************************************************
+ * File: main.c
+ *
+ * Copyright (c) 2019 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Author: Ford Peprah <ford@squareup.com>
+ *
+ * Created: March 12th, 2019
+ * Modified: March 12th, 2019
+ *****************************************************************************/
+
+#include <SEGGER_RTT.h>
+
+#include <string.h>
+
+#define FOREVER  for(;;)
```

Comment:
NIT: doesn't seem to be used

Reply:
Woops, yah.  Copied it all from the other FW images, haha.

Reply:
Wait, it is used below on line 48.